File: importer-service/src/main/java/com/ems/importerservice/config/AppConfig.java
package com.ems.importerservice.config;

import org.springframework.boot.context.properties.EnableConfigurationProperties;
import org.springframework.context.annotation.Configuration;
import org.springframework.scheduling.annotation.EnableScheduling;

@Configuration
@EnableScheduling
@EnableConfigurationProperties({EventServiceClientProperties.class, ImportProperties.class})
public class AppConfig {
}

File: payment-service/src/main/kotlin/com/ems/paymentservice/config/AppConfig.kt
package com.ems.paymentservice.config

import org.springframework.boot.context.properties.EnableConfigurationProperties
import org.springframework.context.annotation.Configuration

@Configuration
@EnableConfigurationProperties(
    KafkaTopicsProperties::class,
    OutboxProperties::class,
    ReceiptProperties::class,
    S3Properties::class,
)
class AppConfig
